routes/truckRoutes.js

```diff
@@ -0,0 +1,39 @@
+/*
+** author: Fulin Shen
+** created on: May 21, 2016
+** description: see above
+*/
+
+var express = require('express');
+var router = express.Router();
+var bodyParser = require('body-parser');
+var urlEncoded = bodyParser.urlencoded({ extended: false });
+
+var trucks = require('../trucks');
+
+router.route('/')
+  .get(function(request, response) {
+    response.send(trucks.getTrucks());
+  })
+  .post(urlEncoded, function(request, response) {
+    var newTruck = request.body;
+    if(newTruck) {
+      trucks.addTruck(newTruck);
```

Comment:
Hi, James:

I tested my code today. It seems to work just find. I saw the new truck was added to the webpage without the need to refresh. I also checked in console and didn't see the undefined message. Did I miss anything? 

Fulim